packages/p2p/src/stores/order-store.js

```diff
@@ -194,22 +194,22 @@ export default class OrderStore {
     }
 
     getP2POrderList() {
-        requestWS({ p2p_order_list: 1 }).then(response => {
-            if (response) {
-                if (response.error) {
-                    this.setErrorMessage(response.error.message);
-                } else {
-                    const { p2p_order_list } = response;
-                    const { list } = p2p_order_list || {};
+        const response = this.root_store.general_store.p2p_order_list_response;
```

Comment:
I guess you need to remove the whole Api call from here and using the custom hooks instead